tests/test_github.py

```diff
@@ -51,36 +44,10 @@ def mock__get_github_token_fake() -> Optional[str]:
     return 'token'
 
 
-def mock_should_perform_interactive_slide_out(cmd: str) -> bool:
-    return True
-
-
 class TestMachete:
```

Comment:
`TestGitHub` to keep in line with the conventions?

Reply:
Also found that `TestMachete` is still present in `test_status.py`

Reply:
DONE!